lighthouse-cli/bin.ts

```diff
@@ -78,6 +79,7 @@ const cliFlags = yargs
     'save-artifacts': 'Save all gathered artifacts to disk',
     'list-all-audits': 'Prints a list of all available audits and exits',
     'list-trace-categories': 'Prints a list of all required trace categories and exits',
+    'trace-categories': 'Additional categories to capture with the trace (comma-delimited).',
```

Comment:
how about `addl-trace-categories` ?  to clarify that this doesn't overwrite the defaults